docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -47,6 +47,20 @@ Incidents are raised as they would in any Camunda cluster. Use the variables and
 
 To start a new process instance, click on the **Restart process** button on the start event, or return to the definition view by clicking **View all** on the top banner.
 
+## Rewind a process
+
+After completing part of your process, you can **rewind** to a previous task to test a section of your process. Play will start a new instance and replay your actions up to the selected previous task.
+
+The rewind operation currently does not support the following elements:
+
+- Called processes
+- Timer events that have been triggered automatically without any user action
```

Comment:
Elsewhere in the page it says "Timer events are executed as defined, with the option to skip the wait."
```suggestion
- Timer events that complete without being skipped
```